<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformWidget</class>
 <widget class="QGroupBox" name="TransformWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>297</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="title">
   <string>Transform Settings</string>
  </property>
  <property name="checkable">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="positionLabel">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Vec3Editor" name="positionEditor" native="true">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="rotationLabel">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Rotation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Vec3Editor" name="rotationEditor" native="true">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="scaleLabel">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Vec3Editor" name="scaleEditor" native="true">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Vec3Editor</class>
   <extends>QWidget</extends>
   <header>Vec3Editor.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
